django_mongodb/utils.py

```diff
@@ -25,6 +26,39 @@ def check_django_compatability():
         )
 
 
+def parse_uri(uri, conn_max_age=0, conn_health_checks=False, test=None):
+    """
+    Convert the given uri into a dictionary suitable for Django's DATABASES
+    setting.
+    """
+    uri = parse_uri_mongo(uri)
+    host = None
+    port = None
+    if uri["fqdn"] is not None:
+        # If the fqdn is present, this is a SRV URI and the host is the fqdn.
+        host = f"mongodb+srv://{uri['fqdn']}"
+    else:
+        nodelist = uri.get("nodelist")
+        if len(nodelist) == 1:
+            host, port = nodelist[0]
+        elif len(nodelist) > 1:
+            host = ",".join([f"{host}:{port}" for host, port in nodelist])
```

Comment:
This branch appears to be untested (add an `assert False` and no tests fail). It looks like there may be a bug if port is None appending ":None" to the host.

Also, I see no tests for a non-None PORT.

Reply:
Good catches, thanks! Fixed in 05df78c27580765c503d2a19eeb791ed271fbc9e and 05ba4453f8a2a71b5afbaf1828e57da9531e58ee